client/src/components/ProfileCard/ProfileCard.jsx

```diff
@@ -26,19 +28,24 @@ function ProfileCard({
     setAnchorEl(null);
   };
 
-  useEffect(() => {
-    const fetchUserProfile = async () => {
-      const { data } = await getUserProfile()
-      setProfile(data)
+  const handleImageUpload = (e) => {
```

Comment:
Where is this called? 🤔

Reply:
Oops, I forgot to delete this. I was trying to have an option to upload an image but I didn't finish it.